superset/views/core.py

```diff
@@ -1288,7 +1288,19 @@ def import_dashboards(self):
         """Overrides the dashboards using json instances from the file."""
         f = request.files.get('file')
         if request.method == 'POST' and f:
-            dashboard_import_export.import_dashboards(db.session, f.stream)
+            try:
+                dashboard_import_export.import_dashboards(db.session, f.stream)
+            except DatabaseNotFound as e:
+                flash(_(
+                    'Cannot import dashboard: {}.\n'
+                    'Make sure to create the database before '
+                    'importing the dashboard.',
```

Comment:
For `i18n` strings, the proper formatting is as documented here https://pythonhosted.org/Flask-Babel/#gettext-functions

Reply:
My bad, I gave a bad example when reviewing this initially.

Reply:
>  the proper formatting

thanks! fixed.

> My bad

not at all. My original code had the same problem.